src/components/site/PageHeader.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+
+const PageHeader = (props) => (
+	<div className="dg_page-header">
+		<div className="dg_page-header__box ">
+			<div className="col-12 col-lg-5 col-xl-6">
+				<span className="dg_page-header__pre-title">{props.preTitle !== undefined ? props.preTitle : null}</span>
+				<span className="dg_page-header__title">{props.title} </span>
+			</div>
+			<div className="col-12 col-lg-7 col-xl-6">
+				<p>{props.deck !== undefined ? props.deck : null}</p>
+			</div>
+		</div>
+	</div>
+);
+
+PageHeader.propTypes = {
+	/** PreTitle to be displayed */
```

Comment:
This should probably be futher clarified, but not blocking on this, as I don't think well get this clarity until @danfox01 is back.